README.md

```diff
@@ -2,9 +2,22 @@
 
 # Extension Workshop
 
-Launchpad for why and how to build Firefox extensions.
+Welcome to Extension Workshop, a launchpad for building Firefox extensions! 🚀
 
-## Getting Started
+## Updating Content
```

Comment:
Just a thought: Should not it be something better, because here we are saying `Updating Content` and then below `Content Updates` in Development section.  Can we change it to how to contribute, because this guidelines are general for contributing in any project. Or it should be better to add these guidelines in a separate file `CONTRIBUTING.md` and add the link of the file in README. 
 
WDYT ? @muffinresearch

Reply:
I'll defer this question to @caitmuenster

Reply:
Yes, we can split these out. Let's do that in a separate PR.

Reply:
Yes, we should do this in a separate PR. :)